openlibrary/core/civicrm.py

```diff
@@ -50,7 +49,7 @@ def get_sponsorships_by_contact_id(contact_id, isbn=None):
     data['json'] = json.dumps(data['json'])  # flatten the json field as a string
     r = requests.get(
         lending.config_ia_civicrm_api.get('url', ''),
-        params=urllib.urlencode(data),
+        params=data,
```

Comment:
I'm concerned about removing the `urllib.urleencode` here, I feel like titles w/ encodings and other cases necessitated this addition and I get the feeling this is going to break things.

Reply:
The Python Docs [recommend the use of the Requests module](https://docs.python.org/3/library/urllib.request.html#module-urllib.request) over the Standard Library because it has a much cleaner API and because in handles Unicode issues so much better.

On both Python 2 and Python 3 it favours Unicode over bytes/str and its convince functions just work without the struggles that the Standard Library imposes.